src/UI/WB.UI.Frontend/src/webinterview/store/store.connection.js

```diff
@@ -22,20 +22,16 @@ const connectionStore = {
                 commit('IS_DISCONNECTED', true)
                 // Vue.$api.stop()
 
-                modal.alert({

```

Comment:
why do you use dialog for alerts all time, I think creating alert method will be plus